opm/grid/common/ZoltanPartition.cpp

```diff
@@ -77,7 +77,7 @@ zoltanGraphPartitionGridOnRoot(const CpGrid& cpgrid,
                                                        wells,
                                                        transmissibilities,
                                                        partitionIsEmpty,
-						       edgeWeightsMethod));
+                                                       edgeWeightsMethod));
```

Comment:
no tabs please.

Reply:
Same as above. No tabs in ZoltanPartition.cpp.